codec-dns/src/main/java/io/netty/handler/codec/dns/DefaultDnsRecordEncoder.java

```diff
@@ -140,7 +199,7 @@ private void encodeRawRecord(DnsRawRecord record, ByteBuf out) throws Exception
         out.writeBytes(content, content.readerIndex(), contentLen);
     }
 
-    protected void encodeName(String name, ByteBuf buf) throws Exception {
+    public static void encodeName(String name, ByteBuf buf) throws Exception {
```

Comment:
Not your fault but it'd be very nice if you could add Javadoc for this. 🙏

Reply:
added